gensrc/proto/internal_service.proto

```diff
@@ -639,6 +639,23 @@ message PProcessDictionaryCacheResult {
     optional int64 dictionary_memory_usage = 2;
 };
 
+message PKeyValue {
+    optional string key = 1;
+    optional string value = 2;
+};
+
+message PStreamLoadRequest {
+  optional string db = 1;
+  optional string table = 2;
+  optional string user = 3;
+  optional string passwd = 4;
+  repeated PKeyValue parameters = 5;
```

Comment:
1. `map` needs protobuf 3.x. It seems a bit complicated to use map in `internal_service.proto` because of `syntax = proto2`. This may also need to choose the corrected versions for the client (brpc-java/jprotobuf). I haven't tested it end-to-end. How about leaving it as a TODO, and I will test it before the final release?
2. use PStringPair